cartalyst/sentry/2016-09-05.yml

```diff
@@ -0,0 +1,8 @@
+title:     Null reset codes were allowed
+link:      http://haxx.ml/post/149975211631/how-i-hacked-your-cfp-and-probably-some-other
+cve:       ~
+branches:
+    2.1:
+        time:     2016-10-04 20:18:00
+        versions: ['>=2.1']
```

Comment:
Fixed.

On 2016-09-05 12:11 PM, Marco Pivetta wrote:
> In cartalyst/sentry/2016-09-05.yml
> <https://github.com/FriendsOfPHP/security-advisories/pull/166#discussion_r77540869>:
> 
>> @@ -0,0 +1,8 @@
>> +title:     Null reset codes were allowed
>> +link:      http://haxx.ml/post/149975211631/how-i-hacked-your-cfp-and-probably-some-other
>> +cve:       ~
>> +branches:
>> +    2.1:
>> +        time:     2016-10-04 20:18:00
>> +        versions: ['>=2.1']
> 
> Probably <2.1 (this is basically "which versions are affected")
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/FriendsOfPHP/security-advisories/pull/166/files/781ee411d316e25d7984837a7d326f7873c8860f#r77540869>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABuoTbTFl5bjaherdOqhnR0LztfG4-kks5qnD8ugaJpZM4J1Kbi>.
>